src/Bridge/Symfony/Bundle/Resources/config/doctrine_orm.xml

```diff
@@ -56,6 +56,7 @@
             <argument type="service" id="api_platform.property_accessor" />
             <argument type="service" id="logger" on-invalid="ignore" />
             <argument key="$identifiersExtractor" type="service" id="api_platform.identifiers_extractor.cached" on-invalid="ignore" />
+            <argument key="$nameConverter" type="service" id="api_platform.name_converter" on-invalid="ignore" />
```

Comment:
i'm not sure about this introduction of named arguments, mixing with old style, but i see a precedent has been set already.

Reply:
We checked this and it's fine indeed!

Reply:
i know it works, but it's just a little strange to mix the styles of argument definition :)